src/Dispatcher.php

```diff
@@ -34,10 +34,12 @@ public function dispatch(ActionTypeInterface $action) {
     /** @var \Drupal\sapi\ActionHandlerInterface $instance */
     foreach ($this->SAPIActionHandlerPluginManager->getDefinitions() as $pluginDefinition) {
       try {
-        /** @var \Drupal\sapi\ActionHandlerInterface $instance */
-        $instance = $this->SAPIActionHandlerPluginManager->createInstance($pluginDefinition['id']);
-
-        $instance->process($action);
+        $enabled = \Drupal::config('sapi.action_handlers')->get('enabled');
```

Comment:
Retrieving the Drupal::config need only to be done once, not once per HandlerPlugin.  Consider moving this to outside of the loop.